notifications_utils/take.py

```diff
@@ -1,22 +1,15 @@
 from notifications_utils.field import Field
 
 
-class Take():
-
-    def __init__(self, thing):
-        self.thing = thing
+class Take(str):
 
     @classmethod
     def from_field(cls, field):
-        return cls(str(field))
+        return cls(field)
```

Comment:
does this from_field give us anything now?

as the casting is implicit now, so 

```
Take.from_field(my_field) == Take(my_field)
```

Reply:
Good point. We can lose `.from_field` too. See a072fb2.